setup.cfg

```
@@ -12,8 +70,16 @@ versionfile_build = hveto/_version.py
 tag_prefix =
 parentdir_prefix =
 
+; -- tools ------------------
+
+[aliases]
+test = pytest
```

@duncanmmacleod, would this work instead?

```suggestion
test = python -m pytest
```

---

no, not really, but that is the root of the problem, namely that I removed the functionality for `setup.py test` from `setup.py` itself. I don't think it can be replicated in `setup.cfg`, so I just need to patch the CI to not use it.